rest_framework/fields.py

```diff
@@ -1675,8 +1676,8 @@ def run_child_validation(self, data):
 
 
 class DictField(Field):
-    child = _UnvalidatedField()
-    initial = {}
+    child = _UnvalidatedField()  # type: Field
+    initial = {}  # type: Dict[str, Any]
```

Comment:
Again here, I'm a bit like, can't we just stick with `= {}`. I appreciate the `str` restricts the keys _some_ but... ?